scl/arr/arr.conf

```diff
@@ -0,0 +1,153 @@
+#############################################################################
+# Copyright (c) 2024 Attila Szakacs
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+#
+# Example log:
+# 2024-01-26 17:20:01.5|Info|ImportListSyncService|Import List Sync Completed. Items found: 1, Series added: 0
+
+block parser arr_internal(
+  app()
+  prefix()
+)
+{
+  channel {
+    parser {
+      regexp-parser(
+        patterns('^([^|]*)\|([^|]*)\|([^|]*)\|(?<MESSAGE>(?:.|\n)*)')
+      );
+      date-parser(
+        template("$1")
+        format("%Y-%m-%d %H:%M:%S.%f")
+      );
+    };
+
+    rewrite {
+      set("`app`" value("APP"));
```

Comment:
The "APP" value is not something we've used thus far. Wouldn't it make more sense to use PROGRAM here?

Or ${.app.name} just as app-parser( )does?

Reply:
Thanks, changed it (and some others, that I have recently contributed) to use `$PROGRAM`.